docs/.vitepress/theme/components/Home.vue

```diff
@@ -93,6 +126,50 @@ const quotesOffsetDesktop = computed(() => `translateX(${((activeQuote.value * 6
   </video>
 </div>
 
+<div v-if="!isSubscribed" class="mt-32 md:mt-40 xl:mt-60 px-6 md:px-12 content-container">
+  <h2 class="text-center">stay informed About Kitops</h2>
```

Comment:
stay => Stay. It should start with uppercase

Reply:
its not a typo :D its a decision. The font renders uppercase and lowercase completely different, so we make some "play" with those to get the style that we want. Example: https://github.com/jozu-ai/kitops/blob/3c9e7aa29ebe399609b6df4488849a2faeaf9585/docs/.vitepress/theme/components/Home.vue#L200

And if i make that text capitalized, it looks like this, which is no bueno:

![image](https://github.com/user-attachments/assets/4ba6ab59-5d3d-4fa5-8b2a-944aa93af389)